Services/include/general.h

```diff
@@ -25,7 +25,11 @@
 
 SAT_returnState start_general_service(void);
 
-typedef enum { REBOOT = 0 } General_Subtype; // shared with EPS!
+typedef enum {
+    REBOOT = 0,
+    GET_UHF_WATCHDOG_TIMEOUT,
+    SET_UHF_WATCHDOG_TIMEOUT,
+} General_Subtype; // shared with EPS!
```

Comment:
I was probably the one to leave that comment in, but could you remove it since we're here. Also, we should keep the style consistent, either removing the =0 on reboot or adding explicit values to the new ones

Reply:
I'll add explicit values to help people make ground services.